### PR TITLE
Updated name + added query for job.finishedAt

### PR DESCRIPTION
**Pull request by mitkotak**
1. Create a copy of your desired hpc config and change name to "instant_hpc"
2. Submit job as usual

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -815,8 +865,13 @@ app.post('/job/:jobId/submit', async function (req, res) {
             .where('id = :id', { id:  job.id })
             .set({ queuedAt: job.queuedAt })
             .execute()
-        if (job.hpc == "dummy_hpc") {
+        if (job.hpc == "instant_hpc") {
             job.finishedAt = new Date()
+            await connection.createQueryBuilder()
+            .update(Job)
```
Comment (zimo-xiao) [inline server.ts]: Saving the same Job twice is not necessary. You only need to update the job once. Just set the finishedAt to current date (new Date) at the line before `if (job.hpc == "instant_hpc")` if its `instant_hpc `

Comment (mitkotak) [reply server.ts]: Does this look good now ?

Comment (zimo-xiao) [reply server.ts]: Try change it to this @mitkotak 

```javascript
        var connection = await db.connect()
        job.queuedAt = new Date()
        const updateJobTo: any = { queuedAt: job.queuedAt }
        if (job.hpc == 'instant_hpc') {
            updateJobTo.finishedAt = new Date()
            updateJobTo.isFailed = false
        }
        await connection.createQueryBuilder()
            .update(Job)
            .where('id = :id', { id:  job.id })
            .set(updateJobTo)
            .execute()
```

**Review by mitkotak — COMMENTED** [review]
@zimo-xiao Can you have a look now ?

**Review by zimo-xiao — APPROVED** [review]
lgtm. Will merge

**Pull request merged**